frontend/src/embedding-sdk-bundle/lib/override-requests-for-guest-embeds.ts

```diff
@@ -31,40 +31,47 @@ const URL_PATTERNS = {
  */
 const EMBED_URL_TRANSFORMATIONS: Record<
   string,
-  { url: string; method: "GET" | "POST" }
+  (data: Pick<RequestData, "url" | "method">) => {
+    url: string;
+    method: "GET" | "POST";
+  }
```

Comment:
Why this change from objects to functions that return objects? If it's for easier testing, could you mention it in a comment?